docs/developing/dev_environment/local_proxy_remote_solana.md

```diff
@@ -85,14 +86,15 @@ export SOLANA_URL=http://api.devnet.solana.com/
 export REVISION=latest
 ```
 
-3. Download the `docker-compose` [file](https://github.com/neonlabsorg/proxy-model.py/blob/develop/docker-compose/docker-compose-remote-solana.yml). This file should be placed in the same folder with the `keys/` directory.
+3. Download the `docker-compose` [file](https://github.com/neonlabsorg/neon-proxy.py/blob/b3bd1a298f3a437cb48379f348ed71268af382cc/docker-compose.yml). This file should be placed in the same folder with the `keys/` directory.
 ```bash
-wget https://raw.githubusercontent.com/neonlabsorg/proxy-model.py/develop/docker-compose/docker-compose-remote-solana.yml
+wget https://raw.githubusercontent.com/neonlabsorg/neon-proxy.py/b3bd1a298f3a437cb48379f348ed71268af382cc/docker-compose.yml
 ```
 
 4. Start the local environment.
 ```bash   
 docker-compose -f docker-compose-remote-solana.yml up postgres dbcreation proxy indexer -d
```

Comment:
docker-compose-remote-solana.yml no longer exists in links, it has to be removed from tutorial